baselines/pFedHN/pFedHN/comparison_experiments/conf/base.yaml

```diff
@@ -0,0 +1,32 @@
+---
+dataset:
+  # dataset config
+  data: "cifar10"
+  root: "./data"
+
+model:
+  # model config
+  n_kernels: 16
+  lr: 5e-3
+  wd: 5e-5
+  in_channels: 3
+  out_dim: 10
+  variant: 0
+  beta: 1e-3
+  delta: 1e-3
+  gamma: 0.97
+
+client:
+  # client config
+  num_epochs: 50
+  num_rounds: 5000
+  num_nodes: 50
+  batch_size: 64
+  num_classes_per_node: 2
+  fraction_fit: 0.1
+  min_fit_clients: 5
+  min_available_clients: 5
+
```

Comment:
```suggestion

server_device: cpu

```

Reply:
The code should be ready to support the server running on either CPU or GPU regardless of the device the clients use. Currently the code doesn't work if clients don't use the gpu.

Reply:
Here I have given device as torch.device("cuda:0" if torch.cuda.is_available() else "cpu") and with respect to that I have taken care in the rest in the `server.py`